src/bfbs_gen.h

```diff
@@ -130,17 +131,23 @@ class BaseBfbsGenerator : public BfbsGenerator {
   }
 
  protected:
-  const reflection::Object *GetObject(const reflection::Type *type) const {
-    if (type->index() >= 0 && IsStructOrTable(type->base_type())) {
+  const reflection::Object *GetObject(const reflection::Type *type,
+                                      bool element_type = false) const {
```

Comment:
Can you add a doc comment to this function - it took a second to figure out what `element_type` meant. Same below